src/System.ServiceModel.Http/tests/ServiceModel/BasicHttpBindingTest.cs

```diff
@@ -169,6 +169,7 @@ public static void Name_Property_Sets(string value)
     [Theory]
     [InlineData(null)]
     [InlineData("")]
+    [ActiveIssue(1449)]
```

Comment:
>  Ron's PR is going to pave right over this adding the new Wcf conditional fact that will do what you are requesting in a much more elegant way.

Let's leave it as-is then, and have the lost coverage for a few days; no point in doing extra work